common-requirements.md

```diff
@@ -0,0 +1,29 @@
+## Title
+Common Requirements
+
+## Context
+Many projects are trying to use common code.  There is a shared repository that all the projects access.  This pattern applies if there is a Strong Code Owner [pattern to be written] or if there is weak code ownership, or no Benevolent Sponsor [pattern to be written]. Someone (or some project) wrote the code in the first place and contributed it to the repository. The common code is a small percentage of the overall deliverable from any of the projects.  Each project has its own delivery schedule, set of deliverables and customers.  
+Statement of Problem:  The common code in the shared repository isn’t meeting the needs of all the projects that want to use it.  
+
+## Problem Statement
+The common code in the shared repository isn’t meeting the needs of all the projects that want to use it.  
```

Comment:
Do we need to specify in the problem statement why the common code isn't meeting the needs? I.e., ... due to conflicting requirements from the participating projects.